src/emmetHelper.ts

```diff
@@ -169,6 +171,10 @@ function getCurrentWord(document: TextDocument, position: Position): string {
 	}
 }
 
+function makeCursorsGorgeous(expandedWord: string): string {
+	return expandedWord.replace(/\$\{\d+\}/g, '|').replace(/\$\{\d+:([^\}]+)\}/g, '_$1_');
```

Comment:
Well, apparently you took ownership to the tests.

I added the surroundings, so the placeholder is printed in cursive characters, so the users can see that this value can be changed.